codersdk/aibridge.go

```diff
@@ -56,7 +56,7 @@ type AIBridgeToolUsage struct {
 }
 
 type AIBridgeListInterceptionsResponse struct {
-	Total   int64                  `json:"total"`
+	Count   int64                  `json:"count"`
```

Comment:
It'll need to be changed in test files too. I suggest undoing your changes and then doing an automated rename refactor operation in vscode (which should change it everywhere). Then re-run `make gen`